_config.yml

```diff
@@ -3,8 +3,11 @@
 remote_theme: wet-boew/gcweb-jekyll
 title: "GCWeb, the Canada.ca official theme" # Site name or title
 website: "https://wet-boew.github.io/GCWeb/" # URL of your public facing website
-site:
+global:
   lang: fr
```

Comment:
Can you apply the change to replace the following in the file: "sites/includes/variable-core.liquid":
*  `site.lang` by `site.global.lang` 
*  `site.langDir` by `site.global.langDir`

Reply:
Changed.Thanks